qt-bugs@ issue : N144508
Trolltech task ID : 144508
applied: yes
author: Montel Laurent
Minor typo into qdbuscpp2xml program.

Index: qdbuscpp2xml.cpp
===================================================================
--- tools/qdbus/tools/qdbuscpp2xml/qdbuscpp2xml.cpp	(revision 615143)
+++ tools/qdbus/tools/qdbuscpp2xml/qdbuscpp2xml.cpp	(working copy)
@@ -47,7 +47,7 @@
 extern QDBUS_EXPORT QString qDBusGenerateMetaObjectXml(QString interface, const QMetaObject *mo,
                                                        const QMetaObject *base, int flags);
 
-#define PROGRAMNAME     "dbuscpp2xml"
+#define PROGRAMNAME     "qdbuscpp2xml"
 #define PROGRAMVERSION  "0.1"
 #define PROGRAMCOPYRIGHT "Copyright (C) 2006 Trolltech ASA. All rights reserved."
 
